spec/std/llvm/llvm_spec.cr

```diff
@@ -11,4 +11,27 @@ describe LLVM do
       LLVM.normalize_triple("x86_64-linux-gnu").should eq("x86_64-unknown-linux-gnu")
     end
   end
+
+  it ".default_target_triple" do
+    triple = LLVM.default_target_triple
+    {% if flag?(:darwin) %}
+      triple.should match(/-apple-macosx$/)
+    {% elsif flag?(:linux) %}
+      triple.should match(/-linux-/)
```

Comment:
Fails on triples that miss libc (rel: #12593)

```
Failure/Error: {% if flag?(:darwin) %}

       Expected: "x86_64-suse-linux"
       to match: /-linux-/

     # spec/std/llvm/llvm_spec.cr:17
```

Reply:
I suppose just checking for `-linux` should be fine.
```suggestion
      triple.should match(/-linux/)
```

Reply:
There's a `x86_64-redhat-linux6E` triple (https://github.com/llvm/llvm-project/blob/main/clang/lib/Driver/ToolChains/Gnu.cpp#L2331), I don't know if it's supported but it would pass on `/-linux/`